modular_skyrat/master_files/code/_globalvars/maint_loot_common.dm

```diff
@@ -353,6 +355,7 @@ GLOBAL_LIST_INIT(common_loot, list( //common: basic items
 		/obj/item/food/meat/slab/human/mutant/shadow = 1,
 		/obj/item/food/meat/slab/human/mutant/skeleton = 1,
 		/obj/item/food/meat/slab/human/mutant/slime = 1,
-		/obj/item/food/meat/slab/human/mutant/zombie = 1
+		/obj/item/food/meat/slab/human/mutant/zombie = 1,
+		/obj/item/seeds/kronkus = 1
```

Comment:
```suggestion
		/obj/item/seeds/kronkus = 1,
```

Reply:
the code does not like it if you put a comma on the last item in a list